specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01/bms.json

```diff
@@ -600,6 +600,55 @@
           }
         }
       }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupValidateOperation": {
+      "post": {
+        "tags": [
+          "Validate"
+        ],
+        "description": "Validate operation for specified backed up item. This is a synchronous operation.",
+        "operationId": "Validate_Operation",
```

Comment:
I think we need a better name for this, what this will generate now is a client named ```Validate``` with a method named ```Operation``` which isn't very intuitive.  If this API is to validate an existing backup I would suggest something like ```Backup_Validate```.

Reply:
Ok. The idea here is to validate the feasibility of the operation. Validate Operation can work before triggering the actual operation. 
In example, we try to validate whether the restore operation can happen. API is written generic for all operations.
How about Operation_Validate ?

Reply:
Ah I see, yeah let's go with Operation_Validate.